data.js

```diff
@@ -767,8 +767,17 @@ if (0) {
 			b.push(box);
 			b.push('<span class="k-kanji">' + entry.kanji + '</span><br/>');
 			b.push('<div class="k-eigo">' + entry.eigo + '</div>');
-			b.push('<div class="k-yomi">' + yomi + '</div>');
-			b.push('</td></tr><tr><td>' + nums + '</td></tr></table>');
+			b.push('<div class="k-yomi">' + yomi + '</div>');   
+//			b.push('</td></tr><tr><td>' + nums + '</td></tr></table>');			
+			b.push('</td></tr><tr><td>' + nums + '</td></tr>');			
+            if (rcxMain.config.rtkStories != null && typeof rcxMain.config.rtkStories != 'undefined') {
```

Comment:
We probably don't need to check for typeof undefined here and below since if it's undefined it will be `== null`